README.md

```diff
@@ -314,3 +314,105 @@ Wrap your components like so:
       )}
     />
 ```
+
+## Server-side rendering (SSR) / Next.js
+
+-----
+
+**⚠️ Note:️**This client SDK _can_ be used with server-side rendering (SSR). However, it does require you to make certain changes, as detailed below. First-class support for SSR is something we'd like to look into, but it is not prioritized at the moment. If you'd like to help us improve the support, let us know in an issue/PR or [on Slack](slack.unleash.run)!
+
+The code samples below are adapted from issues [#40](https://github.com/Unleash/proxy-client-react/issues/40#issuecomment-1048761755 "GitHub issue #40") and [#50](https://github.com/Unleash/proxy-client-react/issues/50 "GitHub issue #50"). Refer to these issues for more information and context.
+
+-----
+
+You can use the client for server-side rendering in a few different ways, each of which have their own advantages and drawbacks. The three main ways are:
+
+| Method                                                           | Advantage                                                 | Drawback                                                                                                               |
+|------------------------------------------------------------------|-----------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|
+| Per-request toggle fetching                                      | You'll always get the most up-to-date toggle state.       | You must wait for the client to start and to fetch toggles before rendering.                                           |
+| Global server-side Unleash client                                | You can render pages server-side  with toggles evaluated. | The toggle state can be as old as the defined refresh interval.                                                        |
+| Defer starting the client until the page is rendered client-side | Less work to implement, faster response times.            | The work is pushed to the client side, so the features the end-user see might change after the client is instantiated. |
+
+The examples below are tailored towards Next.js, but you should be able to adopt them to other settings.
+
+If you want to use this client to evaluate features server-side, then you need to provide an alternative implementation of `fetch`, using the configuration's `fetch` property, and a storage provider, using the `storageProvider` property. By default, this client tries to use `window.fetch` to retrieve toggles for evaluation and `window.localStorage` as a storage provider, but `window` isn't available during server-side rendering.
+
+### Prefetching toggles (Next.js)
+
+<!-- Todo: explain what this is and how it works -->
```

Comment:
@Tymek Could you help me with this section? Does this example show how you would fetch toggles on _every request_? And do the two code snippets go together?

I'm pretty sure I've seen something in Next's docs where you could provide an SDK as a global context thing for Next. That should let you start the client and just keep running it in the background, saving you the performance cost of starting a new client on every request. Do you know how you'd do that?

Reply:
If you run custom Next.js server, you can start client once. I put my work on React SSR / Next etc on hold while we still work on [insert buzzword here] project. Let's talk about this tomorrow morning :)